### PR TITLE
Only proxy Prometheus /api

### PR DESCRIPTION
**Pull request by spadgett**
/assign @brancz

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -192,9 +192,11 @@ func (s *Server) HTTPHandler() http.Handler {
 	)
 
 	if s.prometheusProxyEnabled() {
+		// Only proxy requests to the Prometheus API, not the UI.
+		prometheusProxyAPIPath := prometheusProxyEndpoint + "/api/"
```
Comment (brancz) [inline server/server.go]: Does this actually work? Looking at this: https://github.com/openshift/console/blob/eae371ba5f35f1b3b295625e5e5b4f2de4fdd637/pkg/proxy/proxy.go#L112

It looks to me that the resulting request received by Prometheus is going to be something along the lines of `/api/api/v1/query`.

**Comment by coreos-ui** [comment]
The PR image failed.

**Comment by spadgett** [comment]
jenkins push

**Comment by coreos-ui** [comment]
The PR image has been pushed to Quay with tag `pr_308_build_407`

**Comment by spadgett** [comment]
@brancz Verified the image on our shared OpenShift cluster

**Comment by spadgett** [comment]
OLM test failure fixed in #326

**Pull request merged**